test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/PackageItemLoaderTests.cs

```diff
@@ -130,6 +133,9 @@ public async Task EmitsSearchTelemetryEvents()
             Assert.Equal(operationId, page1["ParentId"]);
             Assert.IsType<int>(page1["ResultCount"]);
             Assert.IsType<double>(page1["Duration"]);
+            Assert.IsType<double>(page1["AggregationDuration"]);
+            Assert.IsType<string>(page1["SourceTimingDuration"]);
+            Assert.Equal(1, ((string)page1["SourceTimingDuration"]).Split(',').Select(e => double.Parse(e)).Count());
```

Comment:
Same as above. Is this a JSON array?

Reply:
The tests were written for the old change, will fix them.